specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/AvailableWorkloadProfiles.json

```diff
@@ -0,0 +1,133 @@
+    "https"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.App/locations/{location}/availableManagedEnvironmentsWorkloadProfileTypes": {
+      "get": {
+        "tags": [
+          "AvailableWorkloadProfiles"
+        ],
+        "summary": "Get available workload profiles by location.",
+        "description": "Get all available workload profiles for a location.",
+        "operationId": "AvailableWorkloadProfiles_Get",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/AvailableWorkloadProfilesCollection"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "BillingMeters_Get": {
+            "$ref": "./examples/AvailableWorkloadProfiles_Get.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AvailableWorkloadProfile": {
+      "description": "A premium workload profile.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Region of the workload profile."
+        },
+        "properties": {
+          "description": "Revision resource specific properties",
+          "type": "object",
+          "properties": {
+            "billingMeterCategory": {
+              "description": "Used to map workload profile types to billing meter.",
+              "type": "string",
+              "enum": [
+                "PremiumSkuGeneralCompute",
+                "PremiumSkuMemoryOptimized",
+                "PremiumSkuComputeOptimized"
+              ],
+              "x-ms-enum": {
+                "name": "Category",
+                "modelAsString": true
+              }
+            },
+            "cores": {
+              "type": "integer",
+              "format": "int32",
+              "description": "Number of cores in CPU."
+            },
+            "memoryGiB": {
+              "type": "integer",
+              "format": "int32",
+              "description": "Memory in GB."
+            },
+            "friendlyName": {
```

Comment:
>friendlyName [](http://example.com/codeflow?start=13&length=12)

This kind of property is usually called displayName.

Reply:
Fixed